.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,16 @@
+name: pre-commit
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
```

Comment:
Which version will be setup?

Reply:
Set python3.8.10 as our test environment